src/test/java/bio/terra/service/journal/JournalServiceTest.java

```diff
@@ -201,29 +219,19 @@ private void validateEntries(
       String note,
       Map<String, Object> entryMap) {
     List<JournalEntryModel> results = journalService.getJournalEntries(key, resourceType, 0, 100);
-    JournalEntryModel entryUnderTest = results.get(expectedCount - 1);
     assertThat(
-        "the number of entries matches the expected count.",
-        results.size(),
-        equalTo(expectedCount));
+        "the number of entries matches the expected count.", results, hasSize(expectedCount));
+    JournalEntryModel entryUnderTest = results.get(0);
```

Comment:
This is the main test change in response to the sort order change: the most recent entry is the first rather than the last.